src/Console/Helper.php

```diff
@@ -39,17 +39,17 @@ abstract class Helper
     /**
      * ConsoleIo instance.
      *
-     * @var \Cake\Console\ConsoleIo
+     * @var \Cake\Console\ConsoleIoInterface
      */
-    protected ConsoleIo $_io;
+    protected ConsoleIoInterface $_io;
```

Comment:
This seems like something we can easily automate with rector. I can take care of that.